src/znc.cpp

```diff
@@ -628,6 +629,7 @@ bool CZNC::WriteNewConfig(const CString& sConfigFile) {
     VCString vsLines;
 
     vsLines.push_back(MakeConfigHeader());
+    vsLines.push_back("PidFile = /run/znc/znc.pid");
```

Comment:
We are using a sub folder because per default we are creating a dedicated *znc* user to run znc. But normal user cannot write to */run/* per default. That's why we need a sub folder.